package.json

```diff
@@ -45,9 +45,16 @@
     "@types/socket.io": "^1.4.29",
     "@types/uuid": "^3.0.0",
     "autoprefixer": "^7.1.1",
+    "babel-cli": "^6.26.0",
     "babel-core": "^6.24.1",
     "babel-eslint": "^7.2.3",
     "babel-loader": "^7.0.0",
+    "babel-plugin-add-module-exports": "^0.2.1",
+    "babel-plugin-transform-class-properties": "^6.24.1",
+    "babel-plugin-transform-es2015-modules-commonjs": "^6.26.0",
```

Comment:
This too.

(Just remove from .babelrc plugins and see if everything still works.)